### PR TITLE
doc: add travel budgets to team docs

### PR DESCRIPTION
**Pull request by dadlerj**
FYI @sqs in case you're actively working on the /doc folder

### CODE REVIEW

File: doc/team/graphbook/travel.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Team travel budgets
+
+Sourcegraph is a remote-first company. This means that the traditional corporate hub-and-spoke travel model (the few remote teammates coming to HQ to meet the rest of the team) is not sufficient.
+
+We believe that team get-togethers are invaluable for building cohesiveness across distributed teams, and spreading Sourcegraph culture evenly across the company.
+
+There are four types of company-sponsored (and company-paid) travel.
+
+## Non-transferable (use it, or lose it)
```
Comment (nicksnyder) [inline doc/team/graphbook/travel.md]: The top level heading here feels a bit weird given the proceeding sentence and I am wondering if this needs a top level heading at all?

Maybe just include the body of this section before the "There are four types of company-sponsored (and company-paid) travel." sentence and only have top level headings for the 4 types of events.

File: doc/team/graphbook/travel.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+- A teammate cannot use their team budget for individual travel.
+- A teammate cannot skip the winter holiday party to get a longer stay at the Spring/Summer company-wide event.
+
+## Company-wide events
+
+The first category of travel is company-wide events. Sourcegraph holds two company-wide events each year, each fully covered by the company. These include a winter holiday party (typically in San Francisco, near HQ), and a Spring/Summer event (occasionally overlapping with GopherCon or other conferences).
+
+## Team events
+
+Each team is allocated USD $4K per year per teammate for team events (minimum two members of the team). This is designed to allow each team to hold at least one event every six months (in addition to the company-wide events above). This budget can be used for travel (airfare, trains, etc.), loding (hotels, rental homes, etc.), events, meals and entertainment, and more. Teams are expected to use discretion about what events are appropriate for team outings, and should contract People Ops (@noemi, @dan) with any questions.
```
Comment (nicksnyder) [inline doc/team/graphbook/travel.md]:
```suggestion
Each team is allocated USD $4K per year per teammate for team events (minimum two members of the team). This is designed to allow each team to hold at least one event every six months (in addition to the company-wide events above). This budget can be used for travel (airfare, trains, etc.), lodging (hotels, rental homes, etc.), events, meals and entertainment, and more. Teams are expected to use discretion about what events are appropriate for team outings, and should contract People Ops (@noemi, @dan) with any questions.
```

Also, should we create a GitHub team for people ops?

File: doc/team/graphbook/travel.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+- Dev: Core services
+- Dev: Distribution
+- Dev: Code intelligence
+- Dev: Automation
+- DevRel, Marketing, and Growth
+- Sales
+
+## Individual travel
+
+Each teammate has one full-paid trip per year to Sourcegraph HQ in San Francisco for up to seven days and six nights (or another city, if they choose to visit another teammate or group of teammates in another location) for individual work travel. This would come on top of the existing USD $1K per year budget for professional development.
```
Comment (nicksnyder) [inline doc/team/graphbook/travel.md]: > or another city, if they choose to visit another teammate or group of teammates in another location

If they are visiting other teammates, this doesn't count against the team budget?

File: doc/team/graphbook/travel.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+The purpose of providing one “trip” (versus providing a set dollar budget) is to avoid punishing teammates that live far away from California, and would face disproportionately high airfare costs.
+
+## Proessional development and education
+
+Each teammate has a USD $1K budget for professional development and education, as outlined on our [careers page](https://github.com/sourcegraph/careers#-professional-development), that can be used for travel.
```
Comment (nicksnyder) [inline doc/team/graphbook/travel.md]: > that can be used for travel

This feels a bit underspecified to me. Can a teammate apply this educate budget to their travel to visit teammates? Or just to attend conferences/classes, etc.?

File: doc/team/graphbook/travel.md (hunk 2 above)
Comment (dadlerj) [reply doc/team/graphbook/travel.md]: Your call!

File: doc/team/graphbook/travel.md (hunk 3 above)
Comment (dadlerj) [reply doc/team/graphbook/travel.md]: If they're visiting members of their team, and the team agrees that it should be counted against the team budget, then yes. 

The assumption here is that the team budget usage would be governed by social norms (perhaps team votes if the team wants it?), while the individual budget is entirely flexible. See my comment at https://sourcegraph.slack.com/archives/GP100NRFS/p1571073013022200

Comment (nicksnyder) [reply doc/team/graphbook/travel.md]: Makes sense, thanks for clarifying.

File: doc/team/graphbook/travel.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+There are four types of company-sponsored (and company-paid) travel.
+
+## Company-wide events
+
+The first category of travel is company-wide events. Sourcegraph holds two company-wide events each year, each fully covered by the company. These include a winter holiday party (typically in San Francisco, near HQ), and a Spring/Summer event (occasionally overlapping with GopherCon or other conferences).
+
+## Team events
+
+Each team is allocated USD $4K per year per teammate for team events (minimum two members of the team). This is designed to allow each team to hold at least one event every six months (in addition to the company-wide events above). This budget can be used for travel (airfare, trains, etc.), lodging (hotels, rental homes, etc.), events, meals and entertainment, and more. Teams are expected to use discretion about what events are appropriate for team outings, and should contract People Ops (@noemi, @dan) with any questions.
```
Comment (nicksnyder) [inline doc/team/graphbook/travel.md]:
```suggestion
Each team is allocated USD $4K per year per teammate for team events (minimum two members of the team). This is designed to allow each team to hold at least one event every six months (in addition to the company-wide events above). This budget can be used for travel (airfare, trains, etc.), lodging (hotels, rental homes, etc.), events, meals and entertainment, and more. Teams are expected to use discretion about what events are appropriate for team outings, and should contract @sourcegraph/people-ops with any questions.
```

**Comment by sqs** [comment]
@dadlerj ack, updated in #6402

**Pull request merged**